File: python/3477.fruits-into-baskets-ii.py
class Solution:
    def numOfUnplacedFruits(self, fruits: List[int], baskets: List[int]) -> int:
        unplaced = len(fruits)
        for fruit in fruits:
            for i in range(len(baskets)):
                if fruit <= baskets[i]:
                    unplaced -= 1
                    baskets[i] = 0
                    break
        return unplaced

File: cpp/3477.fruits-into-baskets-ii.cpp
class Solution {
public:
    int numOfUnplacedFruits(vector<int>& fruits, vector<int>& baskets) {
        int n = fruits.size(), m = baskets.size();	    
	int unplaced = n;
	for(auto fruit:fruits){
		for(int i = 0; i < m; i++){
			if(fruit <= baskets[i]){
				baskets[i] = 0;
				unplaced--;
				break;
			}
		}
	}
	return unplaced;
    }
};
